src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -634,6 +642,29 @@ bool SOP_SceneCacheSource::convertObject( const IECore::Object *object, const st
 			}
 		}
 		
+		if ( params.tagGroups )
+		{
+			SceneInterface::NameList tags;
+			scene->readTags( tags, IECore::SceneInterface::LocalTag );
+			for ( SceneInterface::NameList::const_iterator it=tags.begin(); it != tags.end(); ++it )
+			{
+				UT_String tag( *it );
+				if ( tag.multiMatch( params.tagFilter ) )
+				{
+					// replace this special character found in SCC tags that will prevent the group from being created
+					tag.substitute(":","_");
```

Comment:
I wonder if we should explicitly exclude the "ObjectType:*" tags here. Those tags are somewhat "behind the scenes" in that they are created automatically at cache time. We wouldn't want people caching back out "ObjectType_MeshPrimitive" accidentally.